### ISSUE TITLE
Accessibility: Prepare recommendation readout

### ISSUE THREAD

**Issue by fiorella-io** [newjersey/navigator.business.nj.gov#4435]
**Problem Statement:** As a stakeholder and team member of NJ BFS, I want recommendations on how to improve the accessibility of our products and processes, to ensure that Business.NJ and Navigator are AA WCAG compliant.

**Benefit Hypothesis:** 
1. From a product/user perspective - Enable the maximum number of potential users to engage with the products, increasing the total addressable market and avoiding frustrated customers from getting tripped up on accessibility shortcomings. There's also both a legal and moral underpinning to investing in product accessibility.
2. From an internal team perspective - “Not everyone on the team needs to be a web accessibility expert, but the more accessibility knowledge among team members, the greater your chance of success, and the less heavy lifting the accessibility experts will have to do."

**Success Criteria:** Have a meeting discussing observations and recommendations document created. Define next steps 

**Out-of-scope:** Conducting an audit

**Assumptions:** 
- Rough notes doc - https://docs.google.com/document/d/148zzs_CPcHWt_lFezVzZOvwhHuc8nY85OCs_qytRwIo/edit?usp=sharing

**Draft User Stories:** n/a

**Feature Readiness Check List:** 
- [x] 1 page doc of recommendations provided to team
- [x] Schedule 1 hr meeting with EJ, Jess, Design team, ?? discussing Amanda's recommendations
- [x] Following the meeting, identify the next steps and priorities 
- [x] Add details to #4569
- [x] Write up Accessibility Plan based on the readout discussions to use for future planning

**Comment by amandabeall** [comment]
**Notes for Design grooming:**

Notes/Considerations
- https://docs.google.com/document/d/148zzs_CPcHWt_lFezVzZOvwhHuc8nY85OCs_qytRwIo/edit?usp=sharing 

AC
- Create a Mural or 1 page doc that contains information to share during meetings
- Schedule 1 hr meeting with EJ, Jess, Design team, ?? discussing Amanda's findings/observations since joining the team
- Identify the next steps/priorities
- Tickets are written for next steps (if needed)

**Comment by amandabeall** [comment]
The accessibility meeting is scheduled for Monday 10/17 with EJ, Jess, Dillon, Mike (optional), Cary, and Fiorella 

The document I will be reviewing can be found here: https://docs.google.com/document/d/16M25bJlfoDjjwPW_fl7O8EeOzdOslGEB2k8PPtLprfA/edit?usp=sharing

**Comment by amandabeall** [comment]
The accessibility meeting was rescheduled for later this week. We should have next steps determined by Thursday 10/20

**Comment by amandabeall** [comment]
Notes from the accessibility meeting were taken in comments on the doc. A video recording of the meeting discussing next steps is attached

https://user-images.githubusercontent.com/114253061/197588750-49bc8e3f-3e05-45b7-a26b-699935bc769f.mp4

**Comment by amandabeall** [comment]
Based on the conversations yesterday, I have created a document for the team to continue the conversation outside of this sprint. That document can be found here: https://docs.google.com/document/d/1BR_d0RoXz8g5Azv2Fca0PQOrK_QIDExKoF0WSGjyciE/edit#

I am planning to meet with @carye3 and @dsullivan-fearless in the next few days to determine the best way to share this document with the rest of the team.

I have shared the accessibility readout with the team, facilitated discussion, and drafted ideas for the next steps. This ticket is now complete.
